src/Numerics/SoaCartesianTensor.h

```diff
@@ -158,16 +157,15 @@ class SoaCartesianTensor
     size_t nR     = nElec * Npbc; // total number of positions to evaluate
     size_t offset = Nlm * nR;     // stride for v/gx/gy/gz/l
 
-    auto* xyz_ptr          = xyz.data();
-    auto* XYZ_vgl_ptr      = XYZ_vgl.data();
-    auto* norm_factor__ptr = norm_factor_.data();
+    auto* xyz_ptr          = xyz.device_data();
+    auto* XYZ_vgl_ptr      = XYZ_vgl.device_data();
+    auto* norm_factor__ptr = norm_factor_.device_data();
     // TODO: make separate ptrs to start of v/gx/gy/gz/l?
     // might be more readable?
     // or just pass one ptr to evaluateVGL and apply stride/offset inside
 
     PRAGMA_OFFLOAD("omp target teams distribute parallel for \
-                    map(to:norm_factor__ptr[:Nlm]) \
-                    map(to: xyz_ptr[:3*nR], XYZ_vgl_ptr[:5*nR*Nlm])")
```

Comment:
Based on OpenMP spack. All the `to` are no-op because xyz_ptr has been mapped already. I don't think gains are from this change. The old code is actually more readable when offload is turned of.

Reply:
OK. We can take this change. See my analysis https://github.com/QMCPACK/qmcpack/pull/5342#issuecomment-2689467085